domain/schema/model/sql/0020-unit.sql

```diff
@@ -313,3 +295,23 @@ SELECT
     u.password_hash
 FROM application AS a
 LEFT JOIN unit AS u ON a.uuid = u.application_uuid;
+
+CREATE TABLE unit_resolved (
+    unit_uuid TEXT NOT NULL PRIMARY KEY,
+    mode_id INT NOT NULL,
+    CONSTRAINT fk_unit_resolved_unit
+    FOREIGN KEY (unit_uuid)
+    REFERENCES unit (uuid),
+    CONSTRAINT fk_unit_resolved_mode
+    FOREIGN KEY (mode_id)
+    REFERENCES resolve_mode (id)
+);
+
+CREATE TABLE resolve_mode (
+    id INT PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+INSERT INTO resolve_mode VALUES
+(0, 'retry-hooks'),
+(1, 'no-hooks');
```

Comment:
There is another option. None.

Is it expected that the lack of a row in the table is no resolved? Happy with that, if that's the design.

Reply:
Yep!